src/components/pages/block-txs/block-transactions-table-row.tsx

```diff
@@ -31,21 +38,33 @@ const BlockTransactionsTableRow = ({
         />
       </TableCell>
       <TableCell>{timestampFormattedAsDate ? utc : distance}</TableCell>
-      <TableCell className="max-w-40 truncate text-primary hover:brightness-150">
-        <Link
-          href={`/address/${transaction.from}`}
-          className="text-sm font-medium leading-none"
-        >
-          {transaction.from}
-        </Link>
+      <TableCell className="max-w-[14rem] truncate px-4 py-2">
```

Comment:
You need to be super cautious on the markup here and not introduce useless/redondant classnames and tags.

This markup is enough to get the job done:

```typescript
<TableCell className="max-w-40">
  <div className="flex items-center gap-1">
    <Link
      href={`/tx/${transaction.hash}`}
      className="truncate text-sm font-medium leading-none text-primary hover:brightness-150"
    >
      {transaction.hash}
    </Link>
    <CopyButton
      content="Copy Hash to clipboard"
      copy={transaction.hash}
    />
  </div>
</TableCell>
```